{"uid":"f58a869764e1a3e6","name":"test_address_prefix[Test for Subsocial]","fullName":"tests.test_network_parameters.TestNetworkPrefix#test_address_prefix","historyId":"c53843d504acb03fe36feefd0e7a8897","time":{"start":1666960675983,"stop":1666960676863,"duration":880},"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","statusTrace":"self = <tests.test_network_parameters.TestNetworkPrefix object at 0x7fed0d57b250>\nchain = <tests.utils.chain_model.Chain object at 0x7fed0da84af0>\n\n    def test_address_prefix(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:15: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7fed0da84af0>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] Subsocial\n\ntests/utils/chain_model.py:29: TimeoutError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","statusTrace":"self = <tests.test_network_parameters.TestNetworkPrefix object at 0x7fed0d57b250>\nchain = <tests.utils.chain_model.Chain object at 0x7fed0da84af0>\n\n    def test_address_prefix(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:15: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7fed0da84af0>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] Subsocial\n\ntests/utils/chain_model.py:29: TimeoutError","steps":[],"attachments":[{"uid":"4d82a8dbb16c1b28","name":"stdout","source":"4d82a8dbb16c1b28.txt","type":"text/plain","size":54}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_network_parameters"},{"name":"subSuite","value":"TestNetworkPrefix"},{"name":"host","value":"fv-az214-469"},{"name":"thread","value":"1665-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_network_parameters"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"chain","value":"<tests.utils.chain_model.Chain object at 0x7fed0da84af0>"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"17e82530d48f7169","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666960664976,"stop":1666960665935,"duration":959}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":115,"skipped":0,"passed":0,"unknown":0,"total":115},"items":[{"uid":"ff4432a188aa015b","reportUrl":"https://nova-wallet.github.io/test-runner/1793//#testresult/ff4432a188aa015b","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666945599478,"stop":1666945600311,"duration":833}},{"uid":"7463a2efd67d2ffb","reportUrl":"https://nova-wallet.github.io/test-runner/1792//#testresult/7463a2efd67d2ffb","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666932334496,"stop":1666932335335,"duration":839}},{"uid":"4f22ca5aceb53299","reportUrl":"https://nova-wallet.github.io/test-runner/1791//#testresult/4f22ca5aceb53299","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666921621363,"stop":1666921622458,"duration":1095}},{"uid":"7d9e0e66237c308f","reportUrl":"https://nova-wallet.github.io/test-runner/1790//#testresult/7d9e0e66237c308f","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666901961590,"stop":1666901962312,"duration":722}},{"uid":"fe6a499924d65fa0","reportUrl":"https://nova-wallet.github.io/test-runner/1789//#testresult/fe6a499924d65fa0","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666888054544,"stop":1666888055486,"duration":942}},{"uid":"4ee8a82b99c69ee","reportUrl":"https://nova-wallet.github.io/test-runner/1788//#testresult/4ee8a82b99c69ee","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666874332432,"stop":1666874333242,"duration":810}},{"uid":"894ae26782a59192","reportUrl":"https://nova-wallet.github.io/test-runner/1787//#testresult/894ae26782a59192","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666858988892,"stop":1666858990105,"duration":1213}},{"uid":"81dcd473c772ec9a","reportUrl":"https://nova-wallet.github.io/test-runner/1786//#testresult/81dcd473c772ec9a","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666845983311,"stop":1666845984328,"duration":1017}},{"uid":"817e69e6698ef836","reportUrl":"https://nova-wallet.github.io/test-runner/1785//#testresult/817e69e6698ef836","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666834170121,"stop":1666834170915,"duration":794}},{"uid":"23f406125811db61","reportUrl":"https://nova-wallet.github.io/test-runner/1784//#testresult/23f406125811db61","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666814710600,"stop":1666814711389,"duration":789}},{"uid":"865583a60611352f","reportUrl":"https://nova-wallet.github.io/test-runner/1783//#testresult/865583a60611352f","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666800334225,"stop":1666800335374,"duration":1149}},{"uid":"71c91b7acd8a76ac","reportUrl":"https://nova-wallet.github.io/test-runner/1782//#testresult/71c91b7acd8a76ac","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666786044004,"stop":1666786045194,"duration":1190}},{"uid":"fadc6c5277f7a2e3","reportUrl":"https://nova-wallet.github.io/test-runner/1781//#testresult/fadc6c5277f7a2e3","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666771555630,"stop":1666771556417,"duration":787}},{"uid":"82c598026a2e13f","reportUrl":"https://nova-wallet.github.io/test-runner/1780//#testresult/82c598026a2e13f","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666758368731,"stop":1666758369674,"duration":943}},{"uid":"6fb9e41b5e288572","reportUrl":"https://nova-wallet.github.io/test-runner/1779//#testresult/6fb9e41b5e288572","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666743501474,"stop":1666743502534,"duration":1060}},{"uid":"6276126881b88715","reportUrl":"https://nova-wallet.github.io/test-runner/1778//#testresult/6276126881b88715","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666728385152,"stop":1666728385917,"duration":765}},{"uid":"35aa8c296ceb6e2","reportUrl":"https://nova-wallet.github.io/test-runner/1777//#testresult/35aa8c296ceb6e2","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666714182164,"stop":1666714183187,"duration":1023}},{"uid":"92cf56f44f87104f","reportUrl":"https://nova-wallet.github.io/test-runner/1776//#testresult/92cf56f44f87104f","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666699688095,"stop":1666699698560,"duration":10465}},{"uid":"717c4208f40763d","reportUrl":"https://nova-wallet.github.io/test-runner/1775//#testresult/717c4208f40763d","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666685397651,"stop":1666685398478,"duration":827}},{"uid":"33a95dcd0aaf81cf","reportUrl":"https://nova-wallet.github.io/test-runner/1774//#testresult/33a95dcd0aaf81cf","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] Subsocial","time":{"start":1666672965479,"stop":1666672966221,"duration":742}}]},"tags":[]},"source":"f58a869764e1a3e6.json","parameterValues":["<tests.utils.chain_model.Chain object at 0x7fed0da84af0>"]}